var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -30,48 +32,42 @@ class Julia(Package):
     version('0.4.5', sha256='cbf361c23a77e7647040e8070371691083e92aa93c8a318afcc495ad1c3a71d9')
     version('0.4.3', sha256='2b9df25a8f58df8e43038ec30bae195dfb160abdf925f3fa193b59d40e4113c5')
 
-    # TODO: Split these out into jl-hdf5, jl-mpi packages etc.
-    variant("cxx", default=False, description="Prepare for Julia Cxx package")
-    variant("hdf5", default=False, description="Install Julia HDF5 package")
-    variant("mpi", default=True, description="Install Julia MPI package")
-    variant("plot", default=False,
-            description="Install Julia plotting packages")
-    variant("python", default=False,
-            description="Install Julia Python package")
-    variant("simd", default=False, description="Install Julia SIMD package")
-    variant("mkl", default=False, description="Use Intel MKL")
+    variant('cxx', default=False, description='Prepare for Julia Cxx package')
+    variant('mkl', default=False, description='Use Intel MKL')
 
     patch('gc.patch', when='@0.4:0.4.5')
     patch('openblas.patch', when='@0.4:0.4.5')
     patch('armgcc.patch', when='@1.0.0:1.1.1 %gcc@:5.9 target=aarch64:')
 
     variant('binutils', default=sys.platform != 'darwin',
-            description="Build via binutils")
+            description='Build via binutils')
```

Comment:
Personally I think we should get rid of this variant: https://spack.readthedocs.io/en/latest/getting_started.html#binutils

Reply:
I can poke at it without it. It may be there for older versions.

Reply:
I did not come across a need for binutils but it could be that a system with a really old version of binutils may need a newer on provided by spack. I took the variant out for now.